dockstore-integration-testing/src/test/resources/dockstore.yml

```diff
@@ -13,6 +13,10 @@ bitbucketClientSecret: <fill me in>
 discourseKey: <fill me in>
 discourseUrl: <fill me in>
 discourseCategoryId: 9
+zenodoClientID: <fill me in>
+zenodoClientSecret: <fill me in>
+zenodoRedirectURI: http://<fill me in>:8080/auth/zenodo.org
+zenodoUrl: https://zenodo.org
```

Comment:
I think this should be called `zenodoBaseUrl` -- you never use it on its own. OTOH, you're following the pattern used with other providers, so maybe it's OK.

Reply:
Left this alone for now; Dockstore zenodo endpoint uses it to construct the zenodo oauth/token URL